packages/CodingStandard/src/Fixer/ControlStructure/ShortIfReturnFixer.php

```diff
@@ -0,0 +1,216 @@
+	{
+		return $tokens->isAllTokenKindsFound([T_IF, T_RETURN, T_IS_IDENTICAL]);
+	}
+
+	/**
+	 * Check if fixer is risky or not.
+	 *
+	 * Risky fixer could change code behavior!
+	 *
+	 * @return bool
+	 */
+	public function isRisky()
+	{
+		return false;
+	}
+
+	/**
+	 * Fixes a file.
+	 *
+	 * @param \SplFileInfo $file A \SplFileInfo instance
+	 * @param Tokens $tokens Tokens collection
+	 */
+	public function fix(\SplFileInfo $file, Tokens $tokens)
+	{
+		foreach ($tokens as $index => $token) {
+			if (! $token->isGivenKind(T_IF)) {
+				continue;
+			}
+			$nextTokenId = $tokens->getNextNonWhitespace($index);
+			$nextToken = $tokens[$nextTokenId];
+
+			// Try to look for condition surrounded by braces, mark its start and end position
+			if ($nextToken->getContent() !== '(') {
+				continue;
+			}
+			$conditionStart = $nextTokenId;
+			$conditionEnd = $this->getConditionEnd($tokens, $conditionStart);
+
+			// Check condition body - it must consist only of "return true;"
+			$tokenId = $tokens->getNextNonWhitespace($conditionEnd);
+			$tokenId = $tokens->getNextNonWhitespace($tokenId);
+			$tokenId = $this->checkConditionBody($tokens, $tokenId);
+			if ($tokenId < 0) {
+				continue;
+			}
+
+			// Check statement after condition - must be "return false;"
+			$tokenId = $this->checkStatementAfterCondition($tokens, $tokenId);
+			if ($tokenId < 0) {
+				continue;
+			}
+			$lastTokenId = $tokenId;
+			$shortReturnTokens = $this->createShortReturnTokens($tokens, $conditionStart, $conditionEnd);
+			$tokenId = $index;
+			while ($tokenId !== $lastTokenId) {
+				$tokens->clearAt($tokenId);
+				$tokenId++;
+			}
+			$tokens->clearAt($tokenId);
+			$tokens->clearEmptyTokens();
+			$tokens->insertAt($index, $shortReturnTokens);
+
+		}
+	}
+
+	/**
+	 * Returns the name of the fixer.
+	 *
+	 * The name must be all lowercase and without any spaces.
+	 *
+	 * @return string The name of the fixer
+	 */
+	public function getName()
+	{
+		return self::class;
+	}
+
+	/**
+	 * Returns the priority of the fixer.
+	 *
+	 * The default priority is 0 and higher priorities are executed first.
+	 *
+	 * @return int
+	 */
+	public function getPriority()
+	{
+		return 0;
+	}
+
+	/**
+	 * Returns true if the file is supported by this fixer.
+	 *
+	 * @param \SplFileInfo $file
+	 *
+	 * @return bool true if the file is supported by this fixer, false otherwise
+	 */
+	public function supports(\SplFileInfo $file)
+	{
+		return true;
+	}
+
+	private function getConditionEnd(Tokens $tokens, $conditionStart)
+	{
+		$nextTokenId = $conditionStart;
+		$startingBracesCount = 1;
+		while ($startingBracesCount > 0) {
+			$nextTokenId = $tokens->getNextMeaningfulToken($nextTokenId);
+			$nextToken = $tokens[$nextTokenId];
+			if ($nextToken->getContent() === '(') {
+				$startingBracesCount++;
+			} elseif ($nextToken->getContent() === ')') {
+				$startingBracesCount--;
+			}
+		}
+		$conditionEnd = $tokens->getPrevMeaningfulToken($nextTokenId);
+		return $conditionEnd;
```

Comment:
Instead of doing this manually, you can use [`findBlockEnd`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/ae567ab177676ae1e57b6fac1ab1633dadbad9f6/src/Tokenizer/Tokens.php#L428) method.

I use it like this to find content of `array()`: https://github.com/Symplify/Symplify/blob/4c1a979752e179dac59eb0431d6b33ce0c2ca6af/packages/CodingStandard/src/Tokenizer/TokenSkipper.php#L20-L23